siriuspy/siriuspy/csdevice/opticscorr.py

```diff
@@ -116,17 +116,17 @@ def get_tune_database(acc):
         'Log-Mon':         {'type': 'string', 'value': 'Starting...'},
 
         'DeltaTuneX-SP':   {'type': 'float', 'value': 0, 'prec': 6,
-                            'hilim': 1, 'lolim': -1, 'high': 1, 'low': -1,
-                            'hihi': 1, 'lolo': -1},
+                            'hilim': 5, 'lolim': -5, 'high': 5, 'low': -5,
+                            'hihi': 5, 'lolo': -5},
         'DeltaTuneX-RB':   {'type': 'float', 'value': 0, 'prec': 6,
-                            'hilim': 1, 'lolim': -1, 'high': 1, 'low': -1,
-                            'hihi': 1, 'lolo': -1},
+                            'hilim': 5, 'lolim': -5, 'high': 5, 'low': -5,
+                            'hihi': 5, 'lolo': -5},
         'DeltaTuneY-SP':   {'type': 'float', 'value': 0, 'prec': 6,
-                            'hilim': 1, 'lolim': -1, 'high': 1, 'low': -1,
-                            'hihi': 1, 'lolo': -1},
+                            'hilim': 5, 'lolim': -5, 'high': 5, 'low': -5,
+                            'hihi': 5, 'lolo': -5},
         'DeltaTuneY-RB':   {'type': 'float', 'value': 0, 'prec': 6,
-                            'hilim': 1, 'lolim': -1, 'high': 1, 'low': -1,
-                            'hihi': 1, 'lolo': -1},
+                            'hilim': 5, 'lolim': -5, 'high': 5, 'low': -5,
+                            'hihi': 5, 'lolo': -5},
 
         'ApplyCorr-Cmd':   {'type': 'int', 'value': 0},
```

Comment:
@anacso17, I have recently reviewed the SOFB database and I changed the name of this PV to `ApplyDelta` in SOFB code. It seems a better name, because we need to tell the difference between **Correction** and **Correctors**. In SOFB I always use the word **Delta** to describe correction and the word **Corr** to reference the correctors.

Reply:
Thank you @fernandohds564 ! I changed the PV names (cd98513,   lnls-sirius/machine-applications#77 and lnls-sirius/hla#116) to keep this standard!